web/src/app/views/events-view/events-view.component.html

```diff
@@ -80,19 +80,19 @@ <h4>Adresa bazénu</h4>
         <p> TJ ČZU Praha 6 – Kamýcká 1275, Praha, Suchdol, 165 00</p>
         <h4>Sraz</h4>
         <ul class="list-unstyled">
-          <li>- v 16:30, dole ve stanici metra Dejvická (linka A)</li>
+          <li>- dole ve stanici metra Dejvická (linka A), většinou někdy odpoledne (Konkrétní čas je vždy napsaný u konkrétního termínu v programu.)</li>
```

Comment:
Sjednotil bych styl sdělení v závorkách - podobně jako níže to tu podle mě nemusí být napsané jako věta s velkým počátečním písmenem a tečkou na konci

Reply:
češtinářsky by to bylo myslím správnější s tečkou, ale asi to dává smysl sjednotit